components/library/libsass/Makefile

```diff
@@ -0,0 +1,53 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2017 Andreas Wacknitz
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		libsass
+COMPONENT_VERSION=	3.4.6
+COMPONENT_PROJECT_URL=	http://libsass.org
+COMPONENT_SUMMARY=	LibSass is a C/C++ port of the Sass engine
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_VERSION).tar.gz
+COMPONENT_ARCHIVE_HASH=	\
+	sha256:686bc09cc31bba32915f42a6da5e7d221c07690d4d597e2748cdd02abd3d7c2c
+COMPONENT_ARCHIVE_URL=	https://github.com/sass/libsass/archive/$(COMPONENT_ARCHIVE)
+COMPONENT_FMRI =	library/sass
```

Comment:
Perhaps, library/libsass , if this name is more common ?

Reply:
Ok, done.